### DISCUSSION TITLE
Counterwallet

### DISCUSSION THREAD

**Discussion by ME86-web** [CounterpartyXCP/CIPs#140] [General]
Hi there, 
I was checking my counterwallet (could not sign in but downloaded the 'Free Wallet' on my phone) And I saw that my bitcoin were transfered from my address to another!! Is there something I can do, someone who can help me? If you need additional information please let me know. I am so upset. How is this even possible I had my private 12 english words pasword.

**Comment by adamkrellenstein** [comment]
Unfortunately no one is developing or hosting Counterwallet anymore. For support with Freewallet, see https://freewallet.io. This forum is for general Counterparty questions.
